receiver/pyroscopereceiver/README.md

```diff
@@ -11,6 +11,18 @@ Implements the Pyroscope ingest protocol and conveys the accepted profiles as Op
 
 - `protocols`: sets the application layer protocols that the receiver will serve. See [Supported Protocols](#supported-protocols). Default is http/s on 0.0.0.0:8062 with max request body size of: 5e6 + 1e6.
 - `timeout`: sets the server reponse timeout. Default is 10 seconds.
+- `metrics`: configures the metrics collection for the Pyroscope receiver.
+    - `enable`: enables or disables metrics collection. Default is true.
+    - `exclude_labels`: a list of label names to exclude from the metrics. Available labels are:
```

Comment:
@akvlad I would like to keep `service` on the counters, and remove it from histograms. Maybe we can add an option to exclude the labels from spefiic metrics? Like an optional metric list for each label under `exclude_labels`

Reply:
Ok. I'll think about that.